screen/budgets/src/main/java/com/ivy/budgets/BudgetScreen.kt

```diff
@@ -244,6 +248,25 @@ private fun Toolbar(
                         fontWeight = FontWeight.ExtraBold
                     )
                 )
+
+                val totalRemainingText = when {
```

Comment:
This transformation should be done in the VM. Change the UI state to be `: String?` and do this calculation and formating in the VM